drivers/ethernet/eth_w5500_priv.h

```diff
@@ -76,6 +76,9 @@
 #define W5500_Sn_RX_MEM_START	0x30000
 #define W5500_RX_MEM_SIZE	0x04000
 
+/* Delay for PHY write/read operations (25.6 us) */
+#define W5500_PHY_ACCESS_DELAY		26U
```

Comment:
Is this delay part of datasheet?

Reply:
No, it's just a small delay, as it will block now.(k_busy_sleep) I did some tests (on my Nordic nRF 5340) about the read back of the register. I never had the situation, that the  loop with k_busy_sleep was necessary.

Reply:
Ideally you need to `git commit --amend` your changes for review fixes, not as separate commit.